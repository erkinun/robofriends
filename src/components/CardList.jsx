import React from 'react'
import Card from './Card'

export default ({ robots }) => (
  <div>
    {robots.map((r) => (
      <Card key={r.id} {...r} />
    ))}
  </div>
)
