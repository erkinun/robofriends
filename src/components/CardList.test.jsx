import React from 'react'
import { shallow } from 'enzyme'
import CardList from './CardList'

it('successfully renders Header', () => {
  expect(shallow(<CardList robots={[]} />)).toMatchSnapshot()
})
