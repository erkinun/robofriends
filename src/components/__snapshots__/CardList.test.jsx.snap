// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`successfully renders Header 1`] = `ShallowWrapper {}`;
